server/game/cards/01-Core/EffervescentPrinciple.js

```diff
@@ -4,7 +4,13 @@ class EffervescentPrinciple extends Card {
     // Play: Each player loses half their <A> (rounding down the loss). Gain 1 chain.
     setupCardAbilities(ability) {
         this.play({
-            effect: 'gain a chain and make both players lose half their amber',
+            effect: 'gain 1 chain and make {1} lose {2} amber and {3} lose {4} amber',
+            effectArgs: (context) => [
+                context.player,
+                Math.floor(context.player.amber / 2),
+                context.player.opponent,
+                Math.floor(context.player.opponent.amber / 2)
```

Comment:
This is going to cause an error for 1 player games, you'll need to guard against that.  Yes it's super annoying.